### PR TITLE
Feature - recruitable evaluations

### PR DESCRIPTION
**Pull request by schodevio**
CRUD for recruitable evaluations

### CODE REVIEW

File: app/forms/v2/evaluations/recruitable_create_form.rb [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          state: :draft,
+          template_name: template.name,
+          sections_attributes: sections
+        )
+      end
+
+      def validate_draft!
+        return if @draft.valid?
+
+        raise ErrorResponderService.new(:invalid_record, 422, @draft.errors.full_messages)
```
Comment (michalsz) [inline app/forms/v2/evaluations/recruitable_create_form.rb]: We can merge/refactor it  to 

raise ErrorResponderService.new(:invalid_record, 422, @draft.errors.full_messages) unless @draft.valid?

File: app/forms/v2/evaluations/recruitable_update_form.rb [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+
+        @draft.save!
+      end
+
+      private
+
+      def validate_draft!
+        return if @draft.valid?
+
+        raise ErrorResponderService.new(:invalid_record, 422, @draft.errors.full_messages)
```
Comment (michalsz) [inline app/forms/v2/evaluations/recruitable_update_form.rb]: we can merge it into one line

File: app/forms/v2/evaluations/recruitable_create_form.rb (hunk 1 above)
Comment (schodevio) [reply app/forms/v2/evaluations/recruitable_create_form.rb]: yes we can, but Rubocop will shout about too long line

File: app/forms/v2/evaluations/recruitable_update_form.rb (hunk 2 above)
Comment (schodevio) [reply app/forms/v2/evaluations/recruitable_update_form.rb]: Rubocop will shout about too long line

**Pull request merged**